amf-client/shared/src/test/resources/production/inter-spec-refs/reports/raml08-raml1-datatype.report

```diff
@@ -1,22 +1,22 @@
 Model: file://amf-client/shared/src/test/resources/production/inter-spec-refs/raml08-raml1-datatype/api.raml
-Profile: RAML 1.0
+Profile: RAML 0.8
 Conforms? false
 Number of results: 2
 
 Level: Violation
 
-- Source: http://a.ml/vocabularies/amf/parser#closed-shape
-  Message: Cannot validate unknown node type nodeShape for RAML 0.8
+- Source: http://a.ml/vocabularies/amf/parser#invalid-type-definition
+  Message: Invalid type def object for RAML 0.8
   Level: Violation
-  Target: file://amf-client/shared/src/test/resources/production/inter-spec-refs/raml08-raml1-datatype/datatype.raml#/shape
+  Target: 
   Property: 
-  Position: Some(LexicalInformation([(2,0)-(5,18)]))
+  Position: Some(LexicalInformation([(2,6)-(2,12)]))
   Location: file://amf-client/shared/src/test/resources/production/inter-spec-refs/raml08-raml1-datatype/datatype.raml
 
-- Source: http://a.ml/vocabularies/amf/core#invalid-cross-spec
-  Message: Cannot reference fragments of another spec
```

Comment:
RamlInterSpecRefsReportTest: this test is a raml 08 api that references a raml fragment. Previously the Raml08Plugin allowed parsing raml fragments, and this was adjusted to fix another test suite (RamlBuilderTestCase). This change however causes the raml fragment to be parsed as an external fragment (in the context of a raml 08 main api) and not a raml data type.